plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx

```diff
@@ -72,14 +79,28 @@ export const GroupProfileCard = ({
     kind: 'group',
   });
 
+  const entityMetadataEditUrl = getEntityMetadataEditUrl(group) ?? '#';
+
   const displayName = profile?.displayName ?? name;
-  const emailHref = profile?.email ? `mailto:${profile.email}` : undefined;
+  const emailHref = profile?.email ? `mailto:${profile.email}` : '#';
 
   return (
     <InfoCard
       title={<CardTitle title={displayName} />}
       subheader={description}
       variant={variant}
+      action={
+        <IconButton
+          aria-label="Edit"
+          disabled={!entityMetadataEditUrl}
+          title="Edit Metadata"
+          component={Link}
+          target="_blank"
+          to={entityMetadataEditUrl}
```

Comment:
Ok did you do the `?? '#'` because you wanted to get rid of a typescript error here? In that case just leave `entityMetadataEditUrl` as possibly `undefined`, and do like `to={entityMetadataEditUrl ?? '#'}` here or something. Could also refactor the code entirely such that if `entityMetadataEditUrl` is not set, then the `IconButton` isn't rendered as a `Link` at all. Less hacky but more code, so I'll let you decide. Could do like

```tsx
<IconButton
  aria-label="Edit"
  title="Edit Metadata"
  {...(entityMetadataEditUrl ? {
    component: Link
    to: entityMetadataEditUrl
  } : {
    disabled: true
  )}
>
  <EditIcon />
</IconButton>
```

but that's arguably not super pretty :) Otherwise

```tsx
action={entityMetadataEditUrl ? (
  <IconButton
    aria-label="Edit"
    title="Edit Metadata"
    component={Link}
    to={entityMetadataEditUrl}    
  >
    <EditIcon />
  </IconButton>
) : (
  <IconButton
    aria-label="Edit"
    title="Edit Metadata"
    disabled
  >
    <EditIcon />
  </IconButton>
)
```

or even make a `const action = entityMetadataEditUrl ? ...` above here so the jsx gets a bit less chaotic

Reply:
Yes you're right, I tried to avoid the ts lint error hha, okay will update